src/expr/src/expr/mod.rs

```diff
@@ -79,8 +79,8 @@ pub fn build_from_prost(prost: &ExprNode) -> Result<BoxedExpression> {
     use risingwave_pb::expr::expr_node::Type::*;
 
     match prost.get_expr_type()? {
-        Cast | Upper | Lower | Not | IsTrue | IsNotTrue | IsFalse | IsNotFalse | IsNull
-        | IsNotNull | Neg | Ascii | Abs | Ceil | Floor | Round => build_unary_expr_prost(prost),
```

Comment:
`| Ceil | Floor | Round` why remove these three functions.

Reply:
I mistakenly merged the historical version in while resolving the conflict, I'll fix it